sim/front.py

```diff
@@ -0,0 +1,300 @@
+"""Utilities for using frontends this package.
```

Comment:
I didn't understand the English in this sentence... 😅

Reply:
I don't know, functions for frontends... resolve this conversation if I make sense in the documentation now (I'm open to suggestions if not).